modules/apps/29-fee/keeper/relay.go

```diff
@@ -32,12 +32,13 @@ func (k Keeper) WriteAcknowledgement(ctx sdk.Context, packet ibcexported.PacketI
 	// if there is no registered counterparty address then write acknowledgement with empty relayer address and refund recv_fee.
 	forwardRelayer, _ := k.GetCounterpartyPayeeAddress(ctx, relayer, packet.GetDestChannel())
 
-	ack := types.NewIncentivizedAcknowledgement(forwardRelayer, acknowledgement.Acknowledgement(), acknowledgement.Success())
+	// TODO: this is temporarily hardcoded to true. can we remove/deprecate this bool field. NOTE: should revisit this!!
+	ack := types.NewIncentivizedAcknowledgement(forwardRelayer, acknowledgement, true)
```

Comment:
This is my only concern with using `[]byte` in `WriteAcknowledgement` rather than the result type. But in my opinion, using the result type as an arg for `WriteAcknowledgement` feels wrong. 

I think this should be fine as we continue to decouple the wrapped applications from one another. Do you have any big concerns here @colin-axner? Are you happy to move forward with this?

Reply:
Ah yea. Lets keep for now. We will need to revisit in https://github.com/cosmos/ibc-go/issues/7044#issuecomment-2269240139

Reply:
maybe you can link the issue

Reply:
Aye, will do!